public/js/components/EditorSearchBar.js

```diff
@@ -91,23 +99,41 @@ const EditorSearchBar = React.createClass({
     this.search(query);
   },
 
-  onKeyUp(e) {
+  traverseResultsPrev(shortcut, e) {
+    e.stopPropagation();
+    e.preventDefault();
+
     const ed = this.props.editor;
     const ctx = { ed, cm: ed.codeMirror };
     const { query, index, count } = this.state;
 
+    findPrev(ctx, query);
+    const nextIndex = index == 0 ? count - 1 : index - 1;
+    this.setState({ index: nextIndex });
+  },
+
+  traverseResultsNext(shortcut, e) {
+    e.stopPropagation();
+    e.preventDefault();
+
+    const ed = this.props.editor;
+    const ctx = { ed, cm: ed.codeMirror };
+    const { query, index, count } = this.state;
+
+    findNext(ctx, query);
+    const nextIndex = index == count - 1 ? 0 : index + 1;
+    this.setState({ index: nextIndex });
+  },
+
+  onKeyUp(e) {
     if (e.key != "Enter") {
       return;
     }
 
     if (e.shiftKey) {
-      findPrev(ctx, query);
-      const nextIndex = index == 0 ? count - 1 : index - 1;
-      this.setState({ index: nextIndex });
+      this.traverseResultsPrev(null, e);
```

Comment:
it's lame to have to pass null into a fnc. We should update `traverseResultsPrev` so that you can do `this.traverseResultsPrev(e)`

this would work above
`shortcuts.on("CmdOrCtrl+G", (_, e) => this.traverseResultsNext(e));`